### PR TITLE
Manual commit. Region tags for yaml, yml, and sh

### PR DESCRIPTION
**Pull request by ckim328**
Region tags updated to have the following format: 
~~~
- YAML with metadata name <product-prefix>_<directory-with-underscores>_<filename-without-extension>_<kind>_<metadataName>
- YAML w/out metadata name <product-prefix>_<directory-with-underscores>_<filename-without-extension>_<kind>
- Shell <product-prefix>_<directory-with-underscores>_<filename-without-extension>
~~~

### CODE REVIEW

File: demos/bank-of-anthos-asm-manifests/demo-manifests/frontend-custom-deployment.yml [hunk 1]
```diff
@@ -30,19 +31,18 @@ spec:
       terminationGracePeriodSeconds: 5
       containers:
       - name: front
-        # Temporarily use this container image until https://github.com/GoogleCloudPlatform/bank-of-anthos/pull/235 is merged
```
Comment (askmeegs) [inline demos/bank-of-anthos-asm-manifests/demo-manifests/frontend-custom-deployment.yml:33]: One problem the script has is that it doesn't keep YAML comments :/

**Comment by askmeegs** [comment]
Can we double check if any of the old region tags are used in the docs anywhere?

File: demos/bank-of-anthos-asm-manifests/demo-manifests/frontend-custom-deployment.yml (hunk 1 above)
Comment (ckim328) [reply demos/bank-of-anthos-asm-manifests/demo-manifests/frontend-custom-deployment.yml:33]: oof thats true. For now I can comb through this PR and make sure no other relevant/important comments are removed. I noticed that pyyaml did remove comments, I'll look into using a different module for yaml loading

**Comment by ckim328** [comment]
for now we won't merge, until i get CL ready in tandem 👍

**Comment by ckim328** [comment]
Closing this, as I am working on having the python crawler preserving the comments (besides the google license)

**Pull request closed without merging**